pytest/tests/sanity/rpc_max_gas_burnt.py

```diff
@@ -0,0 +1,63 @@
+"""Test max_gas_burnt_view client configuration.
+
+Spins up two nodes with different max_gas_burnt_view client configuration,
+deploys a smart contract and finally calls a view function against both nodes
+expecting the one with low max_gas_burnt_view limit to fail.
+"""
+
+import sys
+import base58
+import base64
+import json
+
+sys.path.append('lib')
+from cluster import start_cluster
+from utils import load_binary_file
+import transaction
+
+def test_max_gas_burnt_view():
+    nodes = start_cluster(2, 0, 1,
+                          config=None,
+                          genesis_config_changes=[],
+                          client_config_changes={
+                              1: {'max_gas_burnt_view': int(1e10)}
+                          })
+
+    contract_key = nodes[0].signer_key
+    fib_contract = load_binary_file('testdata/fib.wasm')
```

Comment:
Yeah, I'd rather not add new blobs to the repo if possible. https://github.com/near/nearcore/pull/4397 adds a new function to the contract, you can use it as an example. 

The following impl of fib should work with our test contract: 



```
diff --git a/runtime/near-test-contracts/test-contract-rs/src/lib.rs b/runtime/near-test-contracts/test-contract-rs/src/lib.rs
index 27f6a0c82..8e10cc91b 100644
--- a/runtime/near-test-contracts/test-contract-rs/src/lib.rs
+++ b/runtime/near-test-contracts/test-contract-rs/src/lib.rs
@@ -294,6 +294,26 @@ pub unsafe fn write_key_value() {
     value_return(size_of::<u64>() as u64, &result as *const u64 as u64);
 }
 
+#[no_mangle]
+pub unsafe fn fib() {
+    input(0);
+    if register_len(0) != size_of::<u64>() as u64 {
+        panic()
+    }
+    let data = [0u8; size_of::<u64>()];
+    read_register(0, data.as_ptr() as u64);
+    let key = u64::from_le_bytes(data);
+    let result = fib_inner(key).to_le_bytes();
+    value_return(result.len() as u64, result.as_ptr() as u64);
+
+    fn fib_inner(n: u64) -> u64 {
+        match n {
+            0 | 1 => 1,
+            _ => fib_inner(n - 1) + fib_inner(n - 1),
+        }
+    }
+}
+
 #[no_mangle]
 pub unsafe fn write_block_height() {
     let block_height = block_index();
```

Reply:
Thanks. I was trying to get it done in AssemblyScript but Rust is much easier. Done.